api/openapi.gen.json

```diff
@@ -61,6 +61,9 @@
           "body": {
             "type": "string"
           },
+          "fingerprint": {
```

Comment:
Do you think the spec also needs the following in POST /pubkeys ?
```
        "requestBody": {
          "content": {
            "application/json": {
              "schema": {
                "$ref": "#/components/schemas/v1.Pubkey"
              }
            }
          }
```

Reply:
We have an error in the paths.yaml so the payload is currently incorrect. This fixes it: https://github.com/RHEnVision/provisioning-backend/pull/161

Reply:
If we merge both, you should be fine.

Reply:
I think merging #161 and rebasing this PR would be correct.

Reply:
Sure, we need an ACK from you there :)

Reply:
I don't think #161 alone addresses it. @akhil-jha is adding schema to the requestBody for POST, 161 only addresses the response body

Reply:
Oh right, asked @ezr-ondrej to fix this properly.

Reply:
Rebased on top of the input body change in OpenAPI.

Reply:
@akhil-jha did it help, are we able to test this now? :)

Reply:
I'm creating the iqe image now. But i think this should work :)
I'll update it here.